src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';').map((el) => el.split(':'));
```

Comment:
[CODE KNOWLEDGE] - There's no need to create an intermediate array with `map` that you immediately iterate over with `forEach`. You can chain `split` directly with `forEach` to process each style declaration, which would be more efficient and concise.